pkg/keyspace/tso_keyspace_group.go

```diff
@@ -672,6 +680,7 @@ func (m *GroupManager) AllocNodesForKeyspaceGroup(id uint32, desiredReplicaCount
 	if err != nil {
 		return nil, err
 	}
+	m.groups[endpoint.StringUserKind(kg.UserKind)].Put(kg)
 	log.Info("alloc nodes for keyspace group", zap.Uint32("id", id), zap.Reflect("nodes", nodes))
```

Comment:
here we use "id" as the field name of the keyspace group id. Shall we unify the names at the line 179 and line 684?